plugin/diagnostics.py

```diff
@@ -366,7 +368,7 @@ def format_diagnostic(self, diagnostic: Diagnostic) -> str:
 
 class DiagnosticsPresenter(object):
 
-    def __init__(self, window: sublime.Window, documents_state: DocumentsState) -> None:
```

Comment:
`DocumentsState` class also be removed.

Reply:
done https://github.com/sublimelsp/LSP/pull/1069/commits/0717444988be660b0e1ec5b4bd1e72b297451be4